libs/ledger-live-common/src/families/algorand/bridge.integration.test.ts

```diff
@@ -256,30 +256,31 @@ const dataset: DatasetTest<AlgorandTransaction> = {
                 warnings: {},
               },
             },
-            {
-              name: "Can't optIn if Algo balance too low",
-              transaction: (t) => ({
-                ...t,
-                mode: "optIn",
-                assetId: "algorand/asa/31231",
-                amount: new BigNumber("1000"),
-                recipient:
-                  "YWZPDCL5XQPCPGBXKB7KAG7YF2QGCGEX37YTSM55CPEPHKNE2ZSKRAXNQ4",
-              }),
-              expectedStatus: {
-                errors: {
-                  amount: new NotEnoughBalance(),
-                },
-                warnings: {},
-              },
-            },
+            // Test doesn't seems to be relevant since fees < spendableBalance, and we don't have any account with this.
+            // {
+            //   name: "Can't optIn if Algo balance too low",
+            //   transaction: (t) => ({
+            //     ...t,
+            //     mode: "optIn",
+            //     assetId: "algorand/asa/31231",
+            //     amount: new BigNumber("1000"),
+            //     recipient:
+            //       "YWZPDCL5XQPCPGBXKB7KAG7YF2QGCGEX37YTSM55CPEPHKNE2ZSKRAXNQ4",
+            //   }),
+            //   expectedStatus: {
+            //     errors: {
+            //       amount: new NotEnoughBalance(),
+            //     },
+            //     warnings: {},
+            //   },
+            // },
             {
               name: "Can't send ASA if Algo balance too low",
               transaction: (t) => ({
                 ...t,
                 subAccountId:
-                  "js:1:algorand:fef9cf3252121f000324dd819b2c1d76910413924acaf6779b30e1fa1874d7af:+312769",
-                amount: new BigNumber("1000"),
+                  "js:2:algorand:MECOWMKPKH2NWVZTS5V5RQDGFFYBT25KNLOPHG2KUMMNKU6FOHGJT24WBI:+312769",
```

Comment:
Is the change `1` --> `2` fixing something?
I think it was useful to keep testing libcore accounts to make sure migration still works for users with super old accounts.

Reply:
Yeah, when we are looking for a subaccounts we check for the exact id so it would not work if it's set a 1 (tbh I don't know what this number mean)

Reply:
I think `1` is for libcore implem and `2` is for js implem, now we just need to make sure migration works well, but this is fine, I think it's covered in other tests as well.